ts-client/src/amm/index.ts

```diff
@@ -63,7 +63,7 @@ type Opt = {
 };
 
 const getAllPoolState = async (poolMints: Array<PublicKey>, program: AmmProgram) => {
-  const poolStates = (await chunkedFetchMultiplePoolAccount(program, poolMints)) as Array<PoolState>;
```

Comment:
why do we need to convert to any first?

Reply:
seems like an outdated thing i added to bypass typescript compile error. It's not happened again then i'm gonna remove it.